tailwind.config.js

```diff
@@ -4,7 +4,34 @@ module.exports = {
     "./src/**/*.{js,jsx,ts,tsx}",
   ],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        primary: "rgb(var(--color-primary))",
```

Comment:
Will we be able to use primary-1, primary-2 colors? They are different and I see only primary listed in the config

Reply:
Also when using `var` i think it just copy paste values.
You defined variable as `#hex value`, will the `rgb()` function work? I just don't know :)

Reply:
> Will we be able to use primary-1, primary-2 colors? They are different and I see only primary listed in the config

If I understood you correctly on the our video call, you want to name our color styles with a stable name and according to some conditions this name doesn't change, but change the value. Change this value we can in the index.css without renaming this stable name.

Reply:
> Also when using `var` i think it just copy paste values. You defined variable as `#hex value`, will the `rgb()` function work? I just don't know :)

I agree with the copy and paste values, but not in all cases, as some of them have multiple options that should change depending on the conditions. For example, we have a main primary (--color-primary-1), but on hover it changes to a second primary (--color-primary-2), so we change the value of the main variable in index.css, and it automatically changed in Tailwind. Therefore, I suggest change the values of only single variables in the Tailwind config.

And hex work in rgb/rgba, already checked it.

Reply:
cool
so I can use primary-1 and primary-2 in the code?

E.g. can I do

className="bg-primary-1 hover:bg-primary-2"